gadget/validate.go

```diff
@@ -230,31 +240,73 @@ func ensureRolesConsistency(roles map[string]*roleInstance, expectedSeed bool) e
 	return nil
 }
 
+func ensureRolesConsistencyOnModalClassic(roles map[string]*roleInstance) error {
```

Comment:
I don't think we want to start using modal to mean with modes I understand the temptation, but is not great to surface it to users, I'm not really sure classic with modes is great either but it's easier to search (and replace) if needed

Reply:
Understood, changed.